cli/src/ledgers.rs

```diff
@@ -48,14 +48,14 @@ pub fn payments(client: &Client, matches: &ArgMatches) -> Result<()> {
     pager.paginate(iter, |result| match result {
         Ok(op) => {
             println!("ID:             {}", op.id());
-            match op.kind() {
-                &OperationKind::CreateAccount(ref create_account) => {
+            match *op.kind() {
```

Comment:
What's the rationale behind this change?

Reply:
@correlator This is recommended by clippy. Instead of matching on a reference, they recommend deref'ing the enum.